File: eTierMeeting_v2/API/DTO/VW_Production_T1_UPF_Delivery_RecordDTO.cs
using System;
namespace eTierV2_API.DTO
{
    public class VW_Production_T1_UPF_Delivery_RecordDTO
    {
        public DateTime? Plan_End_STC { get; set; }
        public string Line_ID { get; set; }
        public string MO_No { get; set; }
        public string MO_Seq { get; set; }
        public string Model_Name { get; set; }
        public string Article { get; set; }
        public int? Plan_Qty { get; set; }
        public int? STC_Output { get; set; }
        public int? STC_FGIN { get; set; }
        public int? STC_Forward { get; set; }
        public int? Balance { get; set; }
        public string Building { get; set; }
    }
}

File: eTierMeeting_v2/SPA/src/app/_core/_models/production/T1/UPF/vW_Production_T1_UPF_Delivery_RecordDTO.ts
export interface VW_Production_T1_UPF_Delivery_RecordDTO {
  plan_End_STC: string | null;
  line_ID: string;
  mO_No: string;
  mO_Seq: string;
  model_Name: string;
  article: string;
  plan_Qty: number | null;
  stC_Output: number | null;
  stC_FGIN: number | null;
  stC_Forward : number | null;
  balance: number | null;
  building: string;
}
